src/js/content_scripts/gamepad-navigator.js

```diff
@@ -186,4 +189,17 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             modelUpdateTransaction.commit();
         }
     };
+
+    /**
+     *
+     * A listener for the gamepad navigator component to clear the connectivity interval
+     * when the instance of the component is destroyed.
+     *
+     * @param {Number} connectivityIntervalReference - The reference to the gamepad
+     *                                                 connection interval loop.
+     *
+     */
+    gamepad.navigator.clearConnectivityInterval = function (connectivityIntervalReference) {
```

Comment:
Unless you're doing something like checking to make sure the interval is non-empty, you should be able just to use `funcName: "clearInterval"` in an invoker or listener definition and should not need to write a function just for this.